### PR TITLE
The ultimate way to slide into someone's DM's. (CS Major Version)

### PR DESCRIPTION
**Pull request by uawasth2**
Hello from academy.

### CODE REVIEW

**Comment by caiyingshen** [comment]
?????????????????????

**Comment by uawasth2** [comment]
Am I not in your DM's now.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-# CommunityConnect Labs
+# &#x2102;ommunity&#x2102;onnect Labs
```
Comment (chyku) [inline README.md:1]: but why

**Review by chyku — COMMENTED** [review]
why

File: README.md (hunk 1 above)
Comment (uawasth2) [reply README.md:1]: It looks &#x2102;ool

**Comment by caiyingshen** [comment]
:(

**Pull request closed without merging**